planning/behavior_path_planner/autoware_behavior_path_planner_common/src/utils/path_safety_checker/safety_check.cpp

```diff
@@ -604,6 +610,10 @@ std::vector<Polygon2d> getCollidedPolygons(
     const auto & ego_polygon = interpolated_data->poly;
     const auto ego_velocity = std::min(interpolated_data->velocity, max_velocity_limit);
 
+    const double ego_yaw = tf2::getYaw(ego_pose.orientation);
+    const double object_yaw = tf2::getYaw(obj_pose.orientation);
+    if (std::abs(ego_yaw - object_yaw) > yaw_difference_th) continue;
```

Comment:
If you have `ego_yaw = 3.13` and `object_yaw = -3.13`, the difference should be `~0.02` but with this code it will be `6.26` which will always be higher than the threshold.
I believe you should use `autoware::universe_utils::normalizeRadian` to make sure the angle difference is between `-pi` and `pi`.

Reply:
Done @maxime-clem

Reply:
Did you forget to push a commit ? The code appear unchanged :thinking: